server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserMailboxesRoutesTest.java

```diff
@@ -202,6 +204,84 @@ void putShouldReturnNotFoundWithNonExistingUser() throws Exception {
                 .containsEntry("message", "Invalid get on user mailboxes");
         }
 
+        @Test
+        void putShouldThrowWhenMailBoxNameWithDots() throws Exception {
```

Comment:
We tend to use `Mailbox` case and not `MailBox` case

Reply:
not fixed?